src/lf_apply.rs

```diff
@@ -775,19 +740,18 @@ pub(crate) unsafe fn rav1d_loopfilter_sbrow_rows<BD: BitDepth>(
         return;
     }
 
-    let mut uv_off: ptrdiff_t;
+    let mut uv_off: ptrdiff_t = 0;
     let mut level_ptr =
         &(*f).lf.level[((*f).b4_stride * (sby * sbsz >> ss_ver) as isize) as usize..];
-    uv_off = 0;
     for x in 0..(*f).sb128w {
         filter_plane_rows_uv::<BD>(
             f,
             have_top,
             level_ptr,
             (*f).b4_stride,
-            ((*lflvl.offset(x as isize)).filter_uv[1]).as_mut_ptr() as *const [[u16; 2]; 2],
-            &mut *p[1].offset(uv_off as isize),
-            &mut *p[2].offset(uv_off as isize),
+            &(*lflvl.offset(x as isize)).filter_uv[1],
+            p[1][uv_off as usize..].as_mut_ptr().add(p_offset[1]),
+            p[2][uv_off as usize..].as_mut_ptr().add(p_offset[1]),
```

Comment:
```suggestion
            p[1][uv_off as usize + p_offset[1]..].as_mut_ptr(),
            p[2][uv_off as usize + p_offset[1]..].as_mut_ptr(),
```

Reply:
Not applicable anymore since slice is used instead of raw pointer.